### DISCUSSION TITLE
QScrollArea bar min height

### DISCUSSION THREAD

**Discussion by hopegame** [quasarframework/quasar#13666] [General - Components / Directives / etc]
If the height is equal to 50px,Content height is 200px, the scroll bar cannot be smaller?
![企业微信截图_20220610111215](https://user-images.githubusercontent.com/8731021/172982506-6c9899ad-26ec-43be-a9b1-dc1be1cf58c0.png)

Desired results（The following is the system default scroll bar）
![image](https://user-images.githubusercontent.com/8731021/172983052-e8307f0d-1d6e-4c43-9333-8c1d2a2986b0.png)

**Comment by pdanpdan** [comment]
https://github.com/quasarframework/quasar/discussions/13526

**Reply by hopegame** [reply]
If the height is less than 50, the minimum scroll bar is 50, which does not meet the requirements, should be calculated by %

**Comment by pdanpdan** [comment]
Yes, I agree with that - can you please gather the info from these 2 discussions and create an issue?

**Reply by hopegame** [reply]
I submitted #13738, hoping to optimize it. thank you

**Comment by pdanpdan** [comment]
Thank you
